whipper/common/mbngs.py

```diff
@@ -271,8 +275,20 @@ def _getMetadata(release, discid, country=None):
     return discMD
 
 
-# see http://bugs.musicbrainz.org/browser/python-musicbrainz2/trunk/examples/
-#     ripper.py
+def getReleaseMetadata(release_id, discid=None, country=None, record=False):
```

Comment:
Maybe you can add a docstring here.

Reply:
Yes, i have added it but not sure if it's good check it out when you have time.

Reply:
Looks good, maybe use this (slight improvement):

```python
    """
    Return a DiscMetadata object based on MusicBrainz Release ID and Disc ID.
    
    If the disc id is not specified, it will match with any disc that is on
    the release disc-list. Otherwise only returns metadata of one disc in
    release disc-list.
    
    :param release_id: MusicBrainz Release ID
    :type release_id: str
    :param discid: MusicBrainz Disc ID
    :type discid: str or None
    :param country: the country the release was issued in
    :type country: str or None
    :param record: whether to record to disc as a JSON serialization
    :type record: bool
    :returns: a DiscMetadata object based on MusicBrainz Release ID & Disc ID
    :rtype: `DiscMetadata`
    """
```

Reply:
Thank you !!!